frontend-cert/js-projects/js-chat-bot/script.js

```diff
@@ -0,0 +1,19 @@
+console.log("Hello! I'm JSBot, your JavaScript fact guide!");
```

Comment:
It looks like JSbot is introduced twice.
Once on this line and then again on line 7. 
So, I vote we update this line here

```suggestion
console.log("Hello! I am going to provide you with some JavaScript facts!");
```